cmd/finch/virtual_machine_remove_test.go

```diff
@@ -49,19 +52,28 @@ func TestRemoveVMAction_runAdapter(t *testing.T) {
 				logger.EXPECT().Info(gomock.Any()).AnyTimes()
 			},
 		},
+		{
+			name: "should forcibly remove the instance",
+			cmd:  removeCmd,
+			args: []string{
+				"--force",
+			},
+			mockSvc: func(logger *mocks.Logger, creator *mocks.LimaCmdCreator, ctrl *gomock.Controller) {
+				command := mocks.NewCommand(ctrl)
+				creator.EXPECT().CreateWithoutStdio("remove", "--force", limaInstanceName).Return(command)
+				command.EXPECT().CombinedOutput()
+				logger.EXPECT().Info(gomock.Any()).AnyTimes()
```

Comment:
I understand that it was already `Info(gomock.Any()).AnyTimes()` before this PR, but maybe it's a good time for us to assert the exact statements as some changes are made to the logging statements in this PR anyway.

1. We can have more confidence.
2. We can have consistent logs assertions (i.e., most other test cases are doing exact matches if possible).

Reply:
the reason I chose to leave it like this is that these tests are meant to cover the `runAdapter` code, and so it's mainly testing if the `--force` argument is parsed correctly. I don't think changing the logging should impact these tests, as we already cover that in the `run` tests below.